processor/src/main/java/org/mapstruct/ap/internal/model/common/Type.java

```diff
@@ -169,12 +174,15 @@ public String getName() {
     }
 
     /**
-     * String that could be used in generated code to reference to this {@link Type}.
+     * String that could be used in generated code to reference to this {@link Type}. The first time a name
+     * is used (e.g. LocalDateTime has java.time.LocalDate and the org.joda.LocalDateTime) it will be marked to be
+     * imported. So if the java.time variant is used first, this name will be used and java.time.LocalDateTime
+     * will be imported, and the org.joda variant will use the FQN (and will not be imported).
```

Comment:
Maybe the following?

```
Returns a String that could be used in generated code to reference to this {@link Type}.<br>
<p>
The first time a name is used (e. g. for {@code LocalDateTime} there is {@code java.time.LocalDateTime} and {@code org.joda.LocalDateTime}) it will be marked as to be imported. So if the {@code java.time} variant is used first, the {@code java.time.LocalDateTime} will be imported and the {@code org.joda} variant will be used with the FQN (and will not be imported).
```